### PR TITLE
Improve envelope alignment

### PR DESCRIPTION
**Pull request by GretaD**
before
alignenvbefore
after
alignenvelope

### CODE REVIEW

**Review by marcoambrosini — COMMENTED** [review]
there still seems to be more space below in the screenshot, or am I hallucinating? :)

**Comment by GretaD** [comment]
> there still seems to be more space below in the screenshot, or am I hallucinating? :)

there's definitively some more space below. Either this was the result when we worked on it and we didnt notice on the small screen, or i missed something when i rebased. Is there anything missing on css? Can you tell?

File: src/components/Envelope.vue [hunk 1]
```diff
@@ -547,6 +547,9 @@ list-item-style.draft .app-content-list-item-line-two {
 .app-content-list-item-star.icon-starred {
 	display: block;
 }
+.app-content-list-item-icon {
+	display: flex;
```
Comment (marcoambrosini) [inline src/components/Envelope.vue:551]: Maybe this?

```suggestion
	display: flex;
	align-items: center;
```

Comment (GretaD) [reply src/components/Envelope.vue:551]: this doesnt do anything, its already added on the wrapper

**Pull request closed without merging**